platform-api/src/routers/health.py

```
@@ -225,6 +226,46 @@ def _check_optional_secrets(settings: WorkerSettings) -> ServiceCheck:
         message=f"{configured_count} of {total} optional secret(s) configured",
     )
 
+async def _check_discord_webhooks(settings: WorkerSettings) -> ServiceCheck:
+    configured_webhooks = [False, False, False, False]
+
+    async def check_webhook(webhook_role: str):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `check_webhook`

Add return type annotation: `bool`

(ANN202)